react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+export default class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      index: null
+    };
+
+    this.backClick = this.backClick.bind(this);
+    this.forwardClick = this.forwardClick.bind(this);
+    this.dotClick = this.dotClick.bind(this);
+  }
+
+  componentDidMount() {
+    this.intId = setInterval(this.forwardClick(), 3000);
+  }
```

Comment:
`setInterval` needs a function definition, instead of a function call

Reply:
updated!